js/platform.js

```diff
@@ -92,7 +106,42 @@ Platform.prototype.update = function () {
                 this.velocity.x *= -1;
             }
             break;
-
+        case 'endgame':
```

Comment:
Same with the creation of "Goats" in endgame, you don't need to create actual Platforms to animate or display them in endgame. Just draw the images of the platforms instead. Better to encapsulate by keeping endgame's logic out of Goat.js or Platform.js. To get them animating around, you'd update x,y coordinates in update() and then draw accordingly in draw() in endgame